onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.h

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+#include <ctime>
+#include "core/common/logging/logging.h"
+#include "core/framework/op_kernel.h"
+#include "NvInfer.h"
+#include "NvOnnxParser.h"
+
+namespace onnxruntime {
+
+static const int kMaxBatchSize = 16;
+static const int kMaxWorkSpaceSize = 16 << 20;
+
+#define CHECK_CUDA(call)                         \
```

Comment:
>CHECK_CUDA [](start = 8, length = 10)

Can we move this to the .cc file so it's not in global scope?

Reply:
done